templates/theme/index.json-ld

```diff
@@ -0,0 +1,203 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Theme",
+		"fr": "Thème"
+	},
+	"description": {
+		"en": "Theme page template",
+		"fr": "Gabarit de page thème"
+	},
+	"modified": "2020-02-12",
+	"componentName": "theme",
+	"status": "stable",
+	"version": "1.0",
+	"pages": {
+		"examples": [
+			{
+				"title": "[Theme]",
+				"language": "en",
+				"path": "theme-en.html"
+			},
+			{
+				"title": "[Thème]",
+				"language": "fr",
+				"path": "theme-fr.html"
+			}
+		],
+		"docs": [
+			{
+				"title": "Theme page",
+				"language": "en",
+				"path": "theme-doc-en.html"
+			},
+			{
+				"title": "Page de thème",
+				"language": "fr",
+				"path": "theme-doc-fr.html"
+			}
+		]
+	},
+	"dependencies": {
+		"en": [
+			{ "title": "Most requested", "url": "https://wet-boew.github.io/GCWeb/components/gc-most-requested/gc-most-requested-doc-en.html", "component": "gc-most-requested" },
+			{ "title": "Services and information", "url": "https://wet-boew.github.io/GCWeb/components/gc-srvinfo/gc-srvinfo-doc-en.html", "component": "gc-srvinfo" }
+		],
+		"fr": [
+			{ "title": "En demande", "url": "https://wet-boew.github.io/GCWeb/components/gc-most-requested/gc-most-requested-doc-fr.html", "component": "gc-most-requested" },
+			{ "title": "Services et renseignements", "url": "https://wet-boew.github.io/GCWeb/components/gc-srvinfo/gc-srvinfo-doc-fr.html", "component": "gc-srvinfo" }
+		]
+	},
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Theme page",
+				"fr": "Page de thème"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Use this template as the landing page for Government of Canada institutions and organizations.",
+				"fr": "Utilisez ce modèle comme page d'accueil pour les institutions et organismes du gouvernement du Canada."
+			},
+			"guidance": {
+				"en": "TBD",
+				"fr": "TBD"
```

Comment:
If we have the info, update the DTO link to the guidance or point for now to their generic page